src/integrationTest/java/org/opensearch/security/SearchOperationTest.java

```diff
@@ -383,6 +384,7 @@ public class SearchOperationTest {
             new AuditConfiguration(true).compliance(new AuditCompliance().enabled(true))
                 .filters(new AuditFilters().enabledRest(true).enabledTransport(true))
         )
+        .nodeSettings(Map.of(OPENSEARCH_RESOURCE_SHARING_ENABLED, false))
```

Comment:
Why is this necessary here?

Reply:
The feature is enabled by default. Which means that the resource sharing index will be created which caused `shouldGetFieldCapabilitiesForAllIndexes_positive` to fail on `containsExactlyIndices` assertion. The other option to not adding this flag is to add that assertion. However, for the sake of this test, since it is not resource-sharing related, I added a  flag to disable it.

Reply:
I would guess that this is only because the resource sharing index is not marked as hidden. Marking it as hidden should fix this issue without changing the observable behavior.